grisha.pletin/SecondHW/dup.rb

```diff
@@ -0,0 +1,13 @@
+def dup(array)
+  array.map! { |x| x.chars.chunk(&:itself).map(&:first).join }
```

Comment:
Why are you using map! method with bang? What's the difference between `map` and `map!`?